### PR TITLE
added optional argument to load_data and clean_data functions

### PR DESCRIPTION
**Pull request by hanbo-e**
I added a 'path' argument to load_data in data.py. The default value is the same as was hardcoded.
I added a 'make_threshhold' argument to the clean_data function in data.py. The default value is "0.01". Not the argument is given as a string and is later converted to float.

### CODE REVIEW

**Comment by Guli-Y** [comment]
Code looks good. 

Just one consideration for future. We are using this clean_data function before train_test_split. In future if we have new data points or have additional test set, we should still be able to use the clean_data function. We need to be careful about using percentage based transformations before train test split. There are two ways:
1. We keep it like this till we figure out which threshold to use and then update this using 'make' names
2. We can also move 'make' transformation after train test split. But this will take more effort.
